### PR TITLE
changed the spec issues-api

### PR DESCRIPTION
**Pull request by achebanets**

### CODE REVIEW

File: spec/swagger.yaml [hunk 1]
```diff
@@ -2306,20 +2307,15 @@ paths:
...
     get:
-      summary: List all issue states
+      summary: Get Issue States
       description: >-
         Currently the only allowed issue states are `OPENED` and `RESOLVED` but
         Smartling may introduce additional states in future. This request
         returns all current allowed states.
       tags:
-        - Issues
+        - Dictionaries
```
Comment (jones-smartling) [inline spec/swagger.yaml]: I believe this is how it groups the API's, so it should still be under Issues i believe.

File: spec/swagger.yaml [hunk 2]
```diff
@@ -2336,16 +2332,62 @@ paths:
...
+      tags:
+        - Dictionaries
+      produces:
+        - application/json
+      parameters:
+        - name: accountUid
+          in: path
+          required: true
+          type: string
+          description: ''
```
Comment (jones-smartling) [inline spec/swagger.yaml]: description should be provided

File: spec/swagger.yaml [hunk 3]
```diff
@@ -2526,15 +2559,47 @@ paths:
...
+        - name: projectId
+          in: path
+          required: true
+          type: string
+          description: ''
+        - name: issueUid
+          in: path
+          required: true
+          type: string
+          description: ''
```
Comment (jones-smartling) [inline spec/swagger.yaml]: please add description (this applies to all params, fields, etc)

File: spec/swagger.yaml [hunk 4]
```diff
@@ -2545,7 +2610,7 @@ paths:
         as needed before attempting to render it. Maximum length is 4000
         characters.
       tags:
-        - Issues
+        - Comments
```
Comment (jones-smartling) [inline spec/swagger.yaml]: i believe it should be issues still

File: spec/swagger.yaml [hunk 5]
```diff
@@ -2385,14 +2385,15 @@ paths:
 #
 # Issues API endpoints
 #
-  /issues-api/v2/dictionary/issue-types:
+  '/issues-api/v2/dictionary/issue-types':
     get:
-      summary: List all issue types
+      summary: Get Issue Types (DEPRECATED)
       description: >-
         This request returns all current issue types. Smartling may occasionally
```
Comment (jones-smartling) [inline spec/swagger.yaml]: I think the description should say what should be used instead

File: spec/swagger.yaml [hunk 6]
```diff
@@ -2385,14 +2385,15 @@ paths:
...
         add or change the list of allowed issue types.
       tags:
         - Issues
+      deprecated: true
```
Comment (jones-smartling) [inline spec/swagger.yaml]: didnt know they had this, good to know

**Comment by jones-smartling** [comment]
issueapidocs

File: spec/swagger.yaml [hunk 7]
```diff
@@ -2849,23 +2899,17 @@ paths:
...
+      summary: Find Issues
       description: >-
         Returns a list of issues matching specified filter options. You can
         filter based on the date issues were created, target languages, strings,
         issue types and states and the user who opened the issue. Unless
         otherwise specified, request will return a maximum of 30 results. All
         parameters are optional.
       tags:
-        - Issues
+        - Search
```
Comment (jones-smartling) [inline spec/swagger.yaml]: should be Issues

File: spec/swagger.yaml [hunk 8]
```diff
@@ -2946,18 +2986,13 @@ paths:
...
-    post:
+    get:
       description: >-
-        Create a new watcher. Any email address can be added as a watcher to receive notifications about an issue being opened, commented on, or closed.
-      summary: Create watcher
+       Get a list of all watchers.
+      summary: List watchers
       tags:
-        - Issues
+        - Watchers
```
Comment (jones-smartling) [inline spec/swagger.yaml]: should be issues

**Comment by jones-smartling** [comment]
I see the response block is not showing up on the rendered page, that needs to be addressed before its merged

File: spec/swagger.yaml [hunk 9]
```diff
@@ -2385,14 +2385,15 @@ paths:
 #
 # Issues API endpoints
 #
-  /issues-api/v2/dictionary/issue-types:
+  '/issues-api/v2/dictionary/issue-types':
     get:
-      summary: List all issue types
+      summary: Get Issue Types (DEPRECATED)
```
Comment (jones-smartling) [inline spec/swagger.yaml]: i dont think the first API listed should be a deprecated API. This should be moved to the bottom of the APi's for Issues

File: spec/swagger.yaml [hunk 10]
```diff
@@ -2967,36 +3030,46 @@ paths:
...
+          in: query
+          required: false
+          type: string
+          description: Unique identifier of corresponding account
+        - name: limit
+          in: query
+          required: false
+          default: 100
+          type: number
+          description: limit
```
Comment (DmitryBochorishvili) [inline spec/swagger.yaml]: @jones-smartling @hchaps, seems like various apis use different description for `limit` and `offset` parameters. Do we want them to be consistent across all apis?

File: spec/swagger.yaml [hunk 11]
```diff
@@ -3008,51 +3081,33 @@ paths:
           in: path
           required: true
           type: string
+          description:  Unique identifier of corresponding account
```
Comment (DmitryBochorishvili) [inline spec/swagger.yaml]: extra space before `Unique`

File: spec/swagger.yaml [hunk 12]
```diff
@@ -3065,22 +3120,24 @@ paths:
           in: path
           required: true
           type: string
-          description: ''
+          description:  Unique identifier of corresponding account
```
Comment (DmitryBochorishvili) [inline spec/swagger.yaml]: extra space before `Unique`

File: spec/swagger.yaml [hunk 13]
```diff
@@ -3065,22 +3120,24 @@ paths:
...
         - name: watcherUid
           in: path
           required: true
           type: string
-          description: ''
+          description:  Unique identifier of corresponding watcher
```
Comment (DmitryBochorishvili) [inline spec/swagger.yaml]: extra space before `Unique`

File: spec/swagger.yaml [hunk 14]
```diff
@@ -3114,40 +3165,36 @@ paths:
...
           required: true
           type: string
-          description: ''
+          description:  Unique identifier of corresponding project
         - name: issueUid
           in: path
           required: true
           type: string
-          description: ''
+          description:  Unique identifier of corresponding issue
```
Comment (DmitryBochorishvili) [inline spec/swagger.yaml]: extra space before `Unique`

File: spec/swagger.yaml [hunk 15]
```diff
@@ -8050,4 +8106,470 @@ definitions:
...
+                - OPENED
+                - RESOLVED
+  IssueGetResponse:
+    allOf:
+    - $ref: '#/definitions/SuccessResponse'
+    - type: object
+      properties:
+        data:
+          type: object
+          properties:
```
Comment (DmitryBochorishvili) [inline spec/swagger.yaml]: Is possible to extract issue and watcher response properties into a separate item and to reuse it wherever needed? Like for get/find/create issues responses?
See https://github.com/Smartling/api-docs/blob/815634552908f52e001413ec930c452dd7895e7c/spec/swagger.yaml#L7777 as an example

File: spec/swagger.yaml (hunk 4 above)
Comment (jones-smartling) [reply spec/swagger.yaml]: This should be Issues

File: spec/swagger.yaml [hunk 16]
```diff
@@ -2967,36 +3030,46 @@ paths:
...
-          in: body
-          required: true
-          schema:
-            $ref: '#/definitions/WatcherRequest'
+          description: Unique identifier of corresponding account
+        - name: email
+          in: query
+          required: false
+          type: string
+          description: Unique identifier of corresponding account
```
[the remark this answers is not in the document]
Comment (jones-smartling) [reply spec/swagger.yaml]: +1

File: spec/swagger.yaml (hunk 10 above)
Comment (jones-smartling) [reply spec/swagger.yaml]: Might need @hchaps to help standardize the language here, but for now its ok. We wont get consistent across defaults and such and that is ok. My 2 cents.

File: spec/swagger.yaml (hunk 14 above)
Comment (achebanets) [reply spec/swagger.yaml]: Done for all places

File: spec/swagger.yaml (hunk 13 above)
Comment (achebanets) [reply spec/swagger.yaml]: Done

File: spec/swagger.yaml (hunk 12 above)
Comment (achebanets) [reply spec/swagger.yaml]: Done

File: spec/swagger.yaml (hunk 11 above)
Comment (achebanets) [reply spec/swagger.yaml]: Done

File: spec/swagger.yaml (hunk 4 above)
Comment (achebanets) [reply spec/swagger.yaml]: Done

**Pull request merged**